extensions/notebook/src/book/bookTreeView.ts

```diff
@@ -272,22 +272,17 @@ export class BookTreeViewProvider implements vscode.TreeDataProvider<BookTreeIte
 
 	getUntitledNotebookUri(resource: string): vscode.Uri {
 		let untitledFileName: vscode.Uri;
-		if (process.platform === 'win32') {
```

Comment:
I'm confused as to why we needed this before but no longer do?

Reply:
Yeah...before changing this I'd suggest looking up when this was introduced and figuring out if there was a reason for this. It might be covering an edge case or something.

(side note - this is where comments are useful...)

Reply:
I had the platform check when I was opening the entire file path as untitled in my first revision (windows and mac behaved differently) and am not sure what changed now, when I tested with just passing the filename without the whole path, it worked both on windows and mac.